src/PockyBot.NET/Extensions.cs

```diff
@@ -0,0 +1,36 @@
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Options;
+using PockyBot.NET.Configuration;
+using PockyBot.NET.Persistence;
+using PockyBot.NET.Services;
+using PockyBot.NET.Services.Triggers;
+
+namespace PockyBot.NET
```

Comment:
I'd rather put extensions as its own namespace, and make these a `Configuration` static class or something.

Reply:
I disagree. From the perspective of someone using this library, I think the extension methods are core functionality. Adding another namespace will just pollute things. So as it is currently, someone will be able to use it with:

```
using PockyBot.NET;

services.ConfigurePockyBot(Configuration);
```

I'd prefer that most of the core functionality (which is pretty much everything that is declared public, note that a lot of the classes in this pr are internal) be living under the root namespace. This is a pretty standard way of writing libraries. Hopefully that makes sense.

Reply:
That actually sounds sensible. Didn't really think of it that way.